README.md

```diff
@@ -2,55 +2,42 @@
 
 # Project Name
 
-> Description the project.
+> Portfolio: setup and mobile version skeleton.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- html & CSS
+- linters (for checking mistakes or errors)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+Basic knowledge on HTML5
+Fundemental knowledge on Flexbox
 
 ### Setup
+GitHub account
 
 ### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+VS code
+Git(Version Control)
+t
 
 
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [Brenda309](https://github.com/githubhandle)
+- Twitter: [@Brenda](https://twitter.com/twitterhandle)
+- LinkedIn: [Wihogora Brenda](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] Kindly change the link to your actual link, so anyone can contact you.

Like
- GitHub: `[Brenda309](https://github.com/Brenda309)`